docs/papers.yml

```
@@ -2,6 +2,16 @@
 # information to generate the "Research Showcase"
 
 papers:
+  - title: Discovery of a Planar Black Hole Mass Scaling Relation for Spiral Galaxies
+    authors:
+      - Benjamin L. Davis (1)
+      - Zehao Jin (1)
+    affliations:
```

```suggestion
    affliations:
```

I think this is the last remaining issue. I can fix this myself though!